.gitignore

```diff
@@ -2,4 +2,6 @@ lerna-debug.log
 npm-debug.log
 node_modules
 build
-lib
\ No newline at end of file
+lib
+/bower_components
```

Comment:
Is it possible to use web-component-tester and webcomponentsjs from npm instead of bower? I'd like to avoid adding two package managers if possible.